code/jack/short_url/short_url_project/urls.py

```diff
@@ -0,0 +1,22 @@
+"""short_url_project URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/4.1/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('short_url', include('short_url.urls')),
```

Comment:
* It would be better to add trailing `/` to the route for 'short_url' so there is no leading `/` in the route in application url. See attached image.
![image](https://user-images.githubusercontent.com/47562501/198108658-8812ad30-1fa4-4c41-aa78-c5e279035d66.png)